zenodo/modules/github/templates/github/index_base.html

```diff
@@ -110,12 +110,13 @@ <h1 align="center"><i class="fa fa-github"></i> Get started</h1>
       {% set latest_dep = description.depositions[-1] if description.depositions else '' %}
 
         {% if latest_dep %}
-          {{doi_badge(latest_dep.doi)}}
+          {{doi_badge(latest_dep.doi, user_id=current_user.id, repository=repo)}}
         {% endif %}
         <h4 class="repo"><a href="https://github.com/{{repo}}">{{repo}}</a>{% if latest_dep and description.hook %} <i class="fa fa-check" style="color: #5cb85c;"></i>{% endif %}</h4>
 
         <small class="text-muted repo-description"> {{description.description}}</small>
         {% if latest_dep %}
+        {% set doi_url = config.CFG_SITE_SECURE_URL + url_for('zenodo_github.latest_doi', user_id=current_user.id, repository=repo) %}
```

Comment:
Same problem here with ``CFG_SITE_SECURE_URL`` and ``_external/_scheme`` as above.